tests/test_error_message_builder.py

```diff
@@ -0,0 +1,34 @@
+from exasol_error_reporting_python.error_message_builder import \
+    ErrorMessageBuilder
+
+
+def test_message():
+    message = str(ErrorMessageBuilder("E-ERJ-TEST-1").
+                  message("Test message"))
+    assert message == "E-ERJ-TEST-1: Test message"
+
+
+def test_mitigation():
+    message = str(ErrorMessageBuilder("E-ERJ-TEST-1").
+                  message("Something went wrong.").
+                  mitigation("Fix it."))
+    assert message == "E-ERJ-TEST-1: Something went wrong. Fix it."
```

Comment:
Please orient on the output format of the other error-reporting-libararies

https://github.com/exasol/error-reporting-lua/blob/main/doc/user_guide/user_guide.md
https://github.com/exasol/error-reporting-java

Reply:
it seems exactly same as other ones
For this example, the output from the java project is same as follows 
`E-TEST-2: Not enough space on device. Delete something.`

Reply:
Ok, then it fine